.drone.yml

```diff
@@ -142,7 +142,25 @@ pipeline:
       - su-exec www-data ./run.sh --remote --config /var/www/owncloud/apps/encryption/tests/acceptance/config/behat.yml --type cli
     when:
       matrix:
-        TEST_SOURCE: encryption
+        TEST_SOURCE: cli-acceptance
+
+  webui-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - BROWSER=chrome
+      - SELENIUM_HOST=selenium
+      - SELENIUM_PORT=4444
+      - TEST_SERVER_URL=http://server
+      - PLATFORM=Linux
+      - BEHAT_SUITE=${BEHAT_SUITE}
+      - MAILHOG_HOST=email
+    commands:
+      - cd /var/www/owncloud/tests/acceptance/
+      - su-exec www-data ./run.sh --remote --config /var/www/owncloud/apps/password_policy/tests/acceptance/config/behat.yml --type webUI
+    when:
+      matrix:
+        TEST_SUITE: webui-acceptance
```

Comment:
This has TEST_SUITE but the matrix entry uses
```
      TEST_SUITE: webui
      TEST_SOURCE: webui-acceptance
```
the whole file needs to be reviewed to see why it has TEST_SOURCE instead of TEST_SUITE in each place.